[SB-696] [SB-536] Updated the key for WHITELABEL

lkkadiri opened the pull request:
[SB-696](https://jira.sprucelabs.ai/jira/browse/SB-696)
[SB-536](https://jira.sprucelabs.ai/jira/browse/SB-536)

@sprucelabsai/engineers

## Description 
Updated the WHITELABEL env var name
## Type
- [ ] Feature
- [x] Bug
- [ ] Tech debt

## Steps to Test or Reproduce
Updated the WHITELABEL env var name

ryanrd commented on src/skillskit/next/_document.js:
```diff
@@ -8,7 +8,7 @@ export default class MyDocument extends Document {
 		const { auth, config } = store && store.getState()
 
-		let whitelabel = config.WHITELABEL_STYLESHEET_URL
+		let whitelabel = config.WHITELABEL
```
I had some questions re: original story, I'm going to mark this as do_not_merge until we can decide if this is the behavior we want

ryanrd replied:
Is the skill configuration WHITELABEL supposed to:  override the Org wide whitelabel, be included in addition to, or be overridden by?  In this code Org whitelabel stylesheet would replace skill whitelabel stylesheet but I’m not sure if this is the behavior we want.  @liquidg3 / @artisgarbage can you weigh in?

mcdermed commented:
This is working great locally with the changes https://github.com/sprucelabsai/sprucebot-skill-guest-feedback/pull/31

liquidg3 commented on src/skillskit/next/_document.js:
```diff
@@ -31,7 +31,7 @@ export default class MyDocument extends Document {
 	render() {
 		return (
-			<html className="skill">
+			<html className={`skill skill-${this.props.config.SLUG}`}>
```
assume config may not be set (when not authed)

mcdermed replied:
Config is the result of our `pluck()` in `config`. It currently doesn't include properties from the server as far as I can tell

lkkadiri commented:
@liquidg3 @ryanrd 
Added a check to see if config has SLUG and assigning a className if it does and also added the orgWhitelabel stylesheet to override all the others if it exists.
